### DISCUSSION TITLE
what is this shield useful for the README.md  doesn't makes sense to me

### DISCUSSION THREAD

**Discussion by phdean** [badges/shields#9210] [Q&A]
what is this shield useful for the README.md  doesn't makes sense to me

**Comment by danydin** [comment]
This is home to [Shields.io](https://shields.io/), a service for concise, consistent, and legible badges in SVG and raster format, which can easily be included in GitHub readmes or any other web page. The service supports dozens of continuous integration services, package registries, distributions, app stores, social networks, code coverage services, and code analysis services. Every month it serves over 870 million images and is used by some of the world's most popular open-source projects, [VS Code](https://github.com/Microsoft/vscode), [Vue.js](https://github.com/vuejs/vue) and [Bootstrap](https://github.com/twbs/bootstrap) to name a few.
